Movie/MoviesMVP/MovieListViewController/Presenter/MovieListPresenter.swift

```diff
@@ -2,3 +2,68 @@
 // Copyright © RoadMap. All rights reserved.
 
 import Foundation
+
+/// Presenter for MovieList ViewController
+final class MovieListPresenter: MovieListPresenterProtocol {
+    // MARK: Public Properties
+
+    let networkService: NetworkServiceProtocol?
+    let router: RouterProtocol
+    var movies: [Movie]?
+    weak var view: MovieListViewProtocol?
+
+    // MARK: Initializers
+
+    init(
+        view: MovieListViewProtocol,
+        networkService: NetworkServiceProtocol,
+        router: RouterProtocol
+    ) {
+        self.view = view
+        self.networkService = networkService
+        self.router = router
+        getMovies(index: 0)
+    }
+
+    // MARK: Public Methods
+
+    func showDescription(movie: Movie) {
```

Comment:
```suggestion
    func showDescription(movie: Movie) {
```
что б вью ничего максимально не знала что тут происходит какой то show нужно писать название метода типо didSelect onTap и тд